modules/serverless-typescript-context-object-reference.adoc

```diff
@@ -0,0 +1,130 @@
+[id="serverless-typescript-context-object-reference_{context}"]
+= TypeScript context object reference
+
+The `context` object has several properties that can be accessed by the function developer.
+
+[id="serverless-typescript-context-object-reference-log_{context}"]
+== log
+
+Provides a logging object that can be used to write output to the cluster logs. The log adheres to the link:https://getpino.io/#/docs/api[Pino logging API].
+
+.Example log
+[source,javascript]
+----
+Function handle(context) {
+  context.log.info(“Processing customer”);
+  return 'OK';
+}
+----
+
+You can access the function by using the `curl` command to invoke it:
```

Comment:
You haven't shown starting the function first. Should it be?

Reply:
To keep it consistent with the other "Developing X functions" sections, let's leave it out.

As to _why_ we don't do it any other section, I suppose we are relying on [1] to instruct the users how to start it, which helps avoid duplicating this in every "Developing X functions" section.

[1] https://docs.openshift.com/container-platform/4.8/serverless/functions/serverless-functions-getting-started.html#serverless-deploy-func-kn_serverless-functions-getting-started